module/importer/src/Domain/Command/ImportDeletedCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Importer\Domain\Command;
+
+class ImportDeletedCommand implements ImporterCommandInterface
+{
+    /**
+     * @JMS\Type("string")
+     */
+    private string $fileName;
+
+    /**
+     * @JMS\Type("string")
+     */
+    private string $sourceType;
+
+    public function __construct(string $fileName, string $sourceType)
```

Comment:
Shroud by base on importId not on specific data .... 
Dedicated Handler should bas on given impart get required data and make deleting process bas on them